### PR TITLE
add: utility classes

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have added utility classes for the component library. Please review

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -37,14 +37,17 @@
...
 }
 
 .card-icon:hover{
     background-color: var(--light-gray);
 }
 
+.card-shadow{
+    box-shadow: 0 1px 4px 1px #acacac;
+}
+
```
Comment (Maianki) [inline components/card.css:50]: you can use CSS variable here.

File: components/button.css [hunk 2]
```diff
@@ -8,43 +7,19 @@
...
-    color: var(--white);
-}
-
-.warning {
-    background-color: var(--warning);
-    color: var(--warning-text);
+.button-anchor {
+    color: black;
+    background-color: #efefef;
 }
```
Comment (Maianki) [inline components/button.css:16]: you can use CSS variable here. And avoid color names.

**Review by Maianki — APPROVED** [review]
Looks good.

**Review by anik31 — APPROVED** [review]
LGTM

File: components/button.css (hunk 2 above)
Comment (ApurvaSawant11) [reply components/button.css:16]: Done. Thanks

File: components/card.css (hunk 1 above)
Comment (ApurvaSawant11) [reply components/card.css:50]: Updated it.

**Pull request merged**